appveyor.yml

```diff
@@ -3,14 +3,16 @@ environment:
 install:
   - ps: Install-Product node 12
   - 'npm -g install npm@6'
```

Comment:
I guess we can remove this line now

Reply:
Removed in https://github.com/apiaryio/dredd/pull/1531/commits/9abe641a834d0bb9609dae2e2df8c38bd106d8f2.